### ISSUE TITLE
[Accessibility]WAI-ARIA property on not supported in Network page elements that does not have a WAI-ARIA role.

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#271]
In order to use any WAI-ARIA property/states in any element,we need to assign WAI-ARIA role to the element.
